src/blueapi/service/interface.py

```diff
@@ -76,6 +79,27 @@ def stomp_client() -> StompClient | None:
         return None
 
 
+@cache
+def numtracker_client() -> NumtrackerClient | None:
+    conf = config()
+    cxt = context()
+    if conf.numtracker is not None:
+        client = NumtrackerClient(url=conf.numtracker.url, headers={})
+        cxt.run_engine.scan_id_source = _update_scan_num
```

Comment:
Could: Move setup logic into `setup()`

Reply:
I think i am happy keeping it where it is for now, it is easier to track being visually isolated.

Reply:
Nevermind, moved it.